documentation/common/address-example1.yaml

```diff
@@ -6,3 +6,4 @@ spec:
     address: myqueue <2>
     type: queue <3>
     plan: standard-small-queue <4>
+    topic: topic-address <5>
```

Comment:
I find this example a bit confusing mentioning only queue type and plan, and then making a reference to a topic. I feel this should be in a separate example.

Reply:
That's a good point, I think separate examples would be an improvement.

Reply:
@k-wall, @lulf, I will create a new issue to add this content in a new, separate example. Thanks!